documentation/builders/spotify.md

```diff
@@ -0,0 +1,39 @@
+# Spotify Integration
+
+The Spotify integration allows to play music directly from Spotify.
+
+> [!IMPORTANT]
+> You need a Spotify Premium subscription to use this functionality
+
+## Needed credentials
+
+For the spotifyd daemon, you need to provide username and password of your Spotify account
+during installation. This is used to enable the Phoniebox to play music from Spotify.
+
+To control the playback (spotipy), you need to create an app as Spotify developer and provide the
+`client_id` and `client_secret` therefore follow the steps:
```

Comment:
Nitpick: For Builders, we don't need to be so technical (`spotifyd`, `spotipy`) as most of them have never heard of it and would potentially get lost. I'd suggest simplify this.

> To allow your Phoniebox to play music from Spotify, you'll need to provide your Spotify username and password.
To control music playback, you'll also need to create a **Spotify developer app** and get a `client_id` and `client_secret`. Here's how to do it:


We could have docs in the Developers section for more advanced topics of this feature.